### PR TITLE
4607 documentation for vuetify element Image

### PR DESCRIPTION
**Pull request by antonfilichkin**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -2608,50 +2608,54 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
...
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/images/)
 
 - __Java__: _package com.epam.jdi.light.vuetify.elements.common.Image.java_
 
 ```java
+    //@FindBy(css = "#GradientsImage .v-image")
+    @UI("#GradientsImage")
+    public static Image gradientsImage;
+    
     @Test
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2620]: please, use, the final version where tests are with descriptions :)

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -2608,50 +2608,54 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
...
+
+| Method | Description | Return Type | 
+|:---| :--- |:---|
+**alternateText()** | Returns image alternate text if present | String
+**getSourcePath()** | Returns path to source image | String
+**isContain()** | Returns true if image is set to preserve its original aspect ratio | boolean
 **hasGradient()** | Shows that image has gradient | boolean
-**hasLimitedHeight()** | Shows that image has limited height | boolean
-**isLoading()** | Shows that image is loading | boolean
+**hasPlaceholder()** | Shows that image has placeholder (shown while image is loading) | boolean
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2654]: why no imageassert methods?

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (antonfilichkin) [reply source/includes/documentation/_vue_js_elements.md:2654]: Fixed

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (antonfilichkin) [reply source/includes/documentation/_vue_js_elements.md]: Fixed

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -2608,50 +2608,66 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
...
-      image.has().height(290.797);
-      image.has().gradient();
-      }
+        gradientsImage.show();
+        gradientsImage.is().displayed()
+            .has().sourcePath("https://cdn.vuetifyjs.com/images/parallax/material2.jpg")
+            .has().width(516.984)
+            .has().height(290.797)
+            .has().gradient();
     }
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2628]: не понимаю, что случилось((( вот из ветки текущей вьютифай 
image

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 4]
```diff
@@ -2608,50 +2608,66 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
...
-      image.has().sourcePath("https://cdn.vuetifyjs.com/images/parallax/material2.jpg");
-      image.has().width(516.984);
-      image.has().height(290.797);
-      image.has().gradient();
-      }
+        gradientsImage.show();
+        gradientsImage.is().displayed()
+            .has().sourcePath("https://cdn.vuetifyjs.com/images/parallax/material2.jpg")
+            .has().width(516.984)
+            .has().height(290.797)
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2626]: у нас под ширину и высоту в интерфейсах заложены инты, так не работает(

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 4 above)
Comment (antonfilichkin) [reply source/includes/documentation/_vue_js_elements.md]: Fixed the scenario. Updated https://github.com/jdi-testing/jdi-light/issues/4782 to include the task to fix the issue with tests that won't actually execute

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 5]
```diff
@@ -2608,50 +2608,66 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
 
 ### 5.29 Images
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/images/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/images/)
```
[the remark this answers is not in the document]
Comment (antonfilichkin) [reply source/includes/documentation/_vue_js_elements.md:2916]: Fixed

File: source/includes/documentation/_vue_js_elements.md [hunk 6]
```diff
@@ -2913,55 +2913,67 @@ For examples of usage see: [Menus tests](https://github.com/jdi-testing/jdi-ligh
...
+    
+    @Test(description = "Check shown image has gradient")
+    public void gradientsImagesTests() {
+        gradientImage.show();
+        gradientImage.is().displayed()
+            .has().sourcePath("https://cdn.vuetifyjs.com/images/parallax/material2.jpg")
+            .has().width(517)
+            .has().height(291)
+            .has().gradient();
     }
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2934]: test should be modified as we discussed on sync

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 6 above)
Comment (antonfilichkin) [reply source/includes/documentation/_vue_js_elements.md:2935]: Replaced the test

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**